umap/tests/integration/test_datalayer.py

```diff
@@ -59,13 +62,14 @@ def test_should_honour_toZoom(live_server, map, datalayer, page):
     page.goto(f"{live_server.url}{map.get_absolute_url()}#6/48.55/14.68")
```

Comment:
```suggestion
    page.goto(f"{live_server.url}{map.get_absolute_url()}#6/48.55/14.68")
    expect(page).to_have_url(re.compile(r".*#6/48\..+/14\..+"))
```

Would it help given the current failing test?

Reply:
We had the same idea! :)